bundles/org.openhab.binding.fronius/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -106,10 +109,30 @@
 			</channel>
 		</channels>
 
-		<properties>
-			<property name="model"/>
-			<property name="serial"/>
-		</properties>
+		<config-description>
+			<parameter name="deviceId" type="integer">
+				<label>Device ID</label>
+				<description>Specific device identifier</description>
+				<default>0</default>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<!-- Ohmpilot Thing Type -->
+	<thing-type id="ohmpilot">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge"/>
+		</supported-bridge-type-refs>
+		<label>Fronius Ohmpilot</label>
+		<description>Fronius Ohmpilot</description>
+
+		<channels>
+			<channel id="powerrealsum" typeId="ohmpilot_powerreal"/>
+			<channel id="energyrealsumconsumed" typeId="ohmpilot_energy"/>
+			<channel id="temperaturechannel1" typeId="ohmpilot_temperature"/>
```

Comment:
Question just to be check that your channel name is appropriate.
The channel type description is "Temperature of the PT1000". Will `temperaturept1000` be a better name ?

Reply:
These names are the fieldnames of the api of that thing. "Temperature_Channel_1" : 23.899999999999999
I prefer to use the same names/keys as used in the source.
This was also the reason, why I used model and serial instead of modelId and serialNumber.

Reply:
Ok